Assignment 3-JS/sass/style.scss

```diff
@@ -0,0 +1,235 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: sans-serif;
+}
+html {
+  font-size: 80%;
+}
+.container-body {
+  width: 100%;
+  height: 100%;
+  background:linear-gradient(to left,rgb(163, 243, 226),rgb(45, 190, 190),rgb(2, 129, 129));
+  color: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .calender {
+      width: 460px;
+      background-color: #00001a;   
+      box-shadow: 5px 5px 10px black;
+      .month {
+          height: 20%;
+          display: flex;
+          justify-content: space-between;
+          align-items: center;
+          text-align: center;
```

Comment:
Check its usage.

Reply:
It is used to place month name, year and right-left arrows in a single line, and place them in equal space and vertically in center of the container.